src/main/resources/assets/chattriggers/js/moduleProvidedLibs.js

```diff
@@ -154,11 +154,12 @@
     loadClass("org.lwjgl.opengl.GL45");
 
     global.cancel = event => {
-        try {
-            EventLib.cancel(event);
-        } catch (err) {
-            if (event.isCancelable())
-                event.setCanceled(true);
+        if (event instanceof Java.type("com.chattriggers.ctjs.minecraft.listeners.CancellableEvent")) {
```

Comment:
Can we pull out these `Java.type` calls? No need to redo the lookup every time

Reply:
fwiw I meant move it completely out of the function, but after looking at rhino it looks like it does caching on package objects, so I guess it's not that big of a deal